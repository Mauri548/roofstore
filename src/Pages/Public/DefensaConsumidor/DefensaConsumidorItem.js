function DefensaConsumidorItem({ title, children, lastModifiedDate }) {
  return (
    <section className="border-top-footer">
      <div className="row DefensaConsumidor-flex-end padding-none">
        <button className="link-color ProductPage-keyword-information-button DefensaConsumidor-bg-transparent">
          Volver arriba
        </button>
      </div>

      <h4 className="DefensaConsumidor-subtitle m-left-5">{title}</h4>
      {lastModifiedDate && <p className="m-left-5">Última modificación: {lastModifiedDate}</p>}
      <div>&nbsp;</div>
      <div className="DefensaConsumidor-section">
        {children}
      </div>
    </section>
  )
}

export default DefensaConsumidorItem
