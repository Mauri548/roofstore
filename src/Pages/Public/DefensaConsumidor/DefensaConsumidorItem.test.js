import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/react"
import { BrowserRouter } from "react-router-dom"
import DefensaConsumidorItem from "./DefensaConsumidorItem"


describe("DefensaConsumidorPage" , () => {
  test("<DefensaConsumidorItem /> tenga la estructura DOM (incluye classnames) del snapshot", () => {
    const component = render(
      <BrowserRouter>
        <DefensaConsumidorItem title="Title" lastModifiedDate="01/01/2021">
          <p>
            A Paragraph or Many Paragraphs
          </p>
        </DefensaConsumidorItem>
      </BrowserRouter>
    )

    expect(component.container).toMatchSnapshot()
  })
})
