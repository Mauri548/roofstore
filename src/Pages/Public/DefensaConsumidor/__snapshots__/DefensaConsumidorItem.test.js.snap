// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefensaConsumidorPage <DefensaConsumidorItem /> tenga la estructura DOM (incluye classnames) del snapshot 1`] = `
<div>
  <section
    class="border-top-footer"
  >
    <div
      class="row DefensaConsumidor-flex-end padding-none"
    >
      <button
        class="link-color ProductPage-keyword-information-button DefensaConsumidor-bg-transparent"
      >
        Volver arriba
      </button>
    </div>
    <h3
      class="DefensaConsumidor-subtitle m-left-5"
    >
      Title
    </h3>
    <p
      class="m-left-5"
    >
      Última modificación: 
      01/01/2021
    </p>
    <div>
       
    </div>
    <div
      class="DefensaConsumidor-section"
    >
      <p>
        A Paragraph or Many Paragraphs
      </p>
    </div>
  </section>
</div>
`;
